### ISSUE TITLE
Code blocks do not respect dark theming.

### ISSUE THREAD

**Issue by mostghost** [kitschpatrol/yanki-obsidian#22]
Code blocks do not seem to respect if Anki is set to dark mode. Perhaps you could allow Shiki's theming to be set manually in options?

![code card improperly themed](https://github.com/user-attachments/assets/b7158fc1-4b3b-4e9e-8e20-637cfa3cbb87)

**Comment by kitschpatrol** [comment]
Thanks, yes that's annoying.

I wonder if there's a nicer way to handle this than adding an option.

Will have to look into if / how Anki indicates the current theme in the card markup / class hierarchy.

Custom CSS isn't a fix for something as finicky as syntax highlighting, but since this issue relates to the general case of ugly notes, I'll mention that the underlying [Yanki CLI tool _does_ support changing note template CSS](https://github.com/kitschpatrol/yanki?tab=readme-ov-file#subcommand-yanki-style)... exposing this somehow in Yanki Obsidian is on the longer-term to-do list.

**Comment by avinashkanaujiya** [comment]
Yes, it definitely looks ugly, not just codeblocks, especially if you are moving from custom-themed cards to the default card. @kitschpatrol I hope you get around to implementing this because it is really important to me and to people who like looking at pretty things.

**Comment by avinashkanaujiya** [comment]
@kitschpatrol I could not handle the ugliness, so I went ahead and updated the Yanki's note type. Now, tell me, what could actually go wrong by doing this?
